src/dbus_api/api/manager_2_2/methods.rs

```diff
@@ -0,0 +1,11 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use dbus::tree::{MTFn, MethodInfo, MethodResult};
+
+use crate::dbus_api::{api::shared::set_key_shared, types::TData};
+
+pub fn set_key(m: &MethodInfo<MTFn<TData>, TData>) -> MethodResult {
+    set_key_shared(m, true)
```

Comment:
I don't understand here why the second argument is hard-coded to ```true``` here, because I had thought that there were some situations where it would still be undesirable to change the settings on the daemon side.

Reply:
This is part of what I want to talk about in stand up. There are two points to discuss here:
1. The bug that I introduced that is resolved here.
2. Whether we should require a terminal to be present when interactively providing a passphrase over the D-Bus. My initial suggestion of allowing the user to choose through a parameter over the D-Bus was influenced by the errors I was seeing in point #1 which are now resolved.

Reply:
Ok!

Reply:
The answer is that we believe that when the D-Bus is up, altering the settings in stratisd always makes sense. It is only for stratis-min that we wan't to be able to decide whether or not to do this.